import/WebViewPage.qml

```diff
@@ -0,0 +1,26 @@
+import QtQuick 2.0
+import Sailfish.Silica 1.0
+
+Page {
+    id: webViewPage
+
+    property var activeWebView
+
+    property int __sailfish_webviewpage
+
+    states: [
+        State {
+            name: "webViewActive"
+            when: activeWebView != null && activeWebView != undefined && activeWebView.visible && activeWebView.active
+            PropertyChanges {
+                target: pageStack
+                _noGrabbing: activeWebView.moving || activeWebView.pinching || activeWebView.dragging
+            }
+            PropertyChanges {
+                target: webViewPage
+                backNavigation: activeWebView.atXBeginning && !activeWebView.pinching
```

Comment:
I'm not sure about the "!activeWebView.pinching" check here (and the other one above)... it makes a rather horrible grab-flicker if you pinchzoom and then pan left and then attempt to backstep via swipe gesture, when this check is included.